src/HtmlSanitizer/HtmlSanitizer.cs

```diff
@@ -578,6 +578,20 @@ public IHtmlDocument SanitizeDom(string html, string baseUrl = "")
             return dom;
         }
 
+        /// <summary>
+        /// Sanitizes the specified parsed HTML body fragment.
+        /// The Document must have been parsed with CSS support.
```

Comment:
>  The Document must have been parsed with CSS support.

This is not strictly necessary I believe. If the document was not parsed with CSS support and thus does not contain style information of course HtmlSanitizer can't magically re-add it. If you want the sanitization process to drop all style leaving out CSS support might actually be a good thing.

Reply:
How about `If the document has not been parsed with CSS support then all styles will be removed` instead?

Reply:
Pushed that up